js/projectView.js

```diff
@@ -49,27 +49,27 @@ projectView.handleCategoryFilter = function() {
 
 projectView.handleMainNav = function() {
   $('.main-nav').on('click', '.tab', function() {
-    $('.tab-content').hide();
+    // $('.tab-content').hide();
```

Comment:
Right now you're not hiding any of your tab content, so your nav menu doesn't really work.